include/algorithms/util/PeakDetection.hpp

```diff
@@ -56,11 +56,11 @@ class PeakDetection
         }
       }
     }
-    if (sort)
+    if (sortByMag)
     {
-      std::sort(peaks.begin(), peaks.end(), [](auto& left, auto& right) {
+        std::sort(peaks.begin(), peaks.end(), [](auto& left, auto& right) {
         return left.second > right.second;
-      });
+        });
```

Comment:
you can revert these changes as well, and `PeakDetection` no longer has any changes in this PR